src/webapp/azext_webapp/tunnel.py

```diff
@@ -62,8 +65,9 @@ def create_basic_auth(self):
 
     def is_port_open(self):
         is_port_open = False
+        # pylint: disable=no-member
```

Comment:
No need for another disable no-member here if it is already disabled for the class (line 37)